package.json

```
@@ -195,16 +191,20 @@
     "@types/url-parse": "^1.4.3",
     "@types/uuid": "^3.4.4",
     "@types/validator": "^9.4.2",
-    "@typescript-eslint/parser": "^3.10.1",
+    "@typescript-eslint/eslint-plugin": "^3.10.1",
```

```suggestion
    "@typescript-eslint/eslint-plugin": "^4.28.4",
```
Since we are updating the eslint dependencies, what you think about updating this library too? Do you see any side effects?